type/deepmerge.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module 'deepmerge' {
```

Comment:
Unless I misunderstand, does `@types/deepmerge` not already provide this?

Reply:
I was having trouble with `@types/deepmerge` 
https://www.npmjs.com/package/@types/deepmerge

I will experiment a bit more to see if the statement: "deepmerge provides its own type definitions" is true.